.circleci/Makefile

```diff
@@ -1,8 +1,7 @@
-all: latest 8.13 8.12
+all: 8.15 dev
```

Comment:
Running `make` on your machine can build the images locally (if your installed Docker works).